MqttChangeLog.txt

```diff
@@ -7,4 +8,4 @@
 - FEATURE IDAS-20578: Add mosquitto 1.4.4 support to Docker
 
 * Thu Apr 30 2015 Agustin Gonzalez <lucas@tid.es> 1.0.1
-- FEATURE IDAS-20368: Add protocol description (IoTA Manager integration).
+- FEATURE IDAS-20368: Add protocol description (IoT Manager integration).
```

Comment:
Thanks for you interesent in the proyect.

IoTA stands for "IoT Agent" and it's the right spelling for this. Thus, I think this PR is not needed.